pycqed/instrument_drivers/meta_instrument/heterodyne.py

```diff
@@ -159,7 +166,8 @@ def prepare_UHFQC(self):
                     self.auto_seq_loading():
                 self._awg_seq_filename = \
                     st_seqs.generate_and_upload_marker_sequence(
```

Comment:
I actually think this should be replaced by a proper sequence as they are defined in `single_qubit_tek_seq_elts.py` but I aprove of the improvements you have made.